packages/fxa-auth-server/lib/metrics/glean/server_events.ts

```diff
@@ -2,7 +2,7 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-// AUTOGENERATED BY glean_parser v8.1.1. DO NOT EDIT. DO NOT COMMIT.
+// AUTOGENERATED BY glean_parser v7.2.2.dev8+g91d4c811. DO NOT EDIT. DO NOT COMMIT.
```

Comment:
Does the old version of Glean parser have issues with passing these values?

Reply:
Arkadiusz on the data engineering team generated this file using a not yet released (but PR approved) branch of the glean_parser.